server/odc-service/src/main/java/com/oceanbase/odc/service/dispatch/RequestDispatcher.java

```diff
@@ -115,6 +119,22 @@ public String getRequestUrlByRequest(HttpServletRequest request) {
         return uriBuilder.toString();
     }
 
+    @SneakyThrows
```

Comment:
don't add this annotation, we may throw the exceptions if there is any~

Reply:
ok